sympy/printing/latex.py

```diff
@@ -2433,6 +2450,12 @@ def latex(expr, fold_frac_powers=False, fold_func_brackets=False,
     gothic_re_im : boolean, optional
         If set to ``True``, `\Re` and `\Im` is used for ``re`` and ``im``, respectively.
         The default is ``False`` leading to `\operatorname{re}` and `\operatorname{im}`.
+    decimal_separator : string, optional
+        Specifies how to print decimal values using the right separator. The separator
```

Comment:
Perhaps
```
Specifies what separator to use to separate the whole and fractional parts of a
floating point number as in "2.5" (for the default, "period") or `2{,}5`
when "comma" is specified.
```

Reply:
This should also specify that it 'comma' changes the separator for lists, sets, and tuples.

Reply:
Thank you! I will update the pull request with these changes!

Reply:
You might also want to add a  warning/disclaimer how the representation of lists, sets, tuples (and whatever else you decide to modify) changes when the ``comma`` decimal separator is used as in using ``semicolon`` to separate elements instead of ``comma``.

Reply:
Also, I think the single backticks work too for numbers